### PR TITLE
Adds 'none' as a valid argument for audit-level

### PR DESCRIPTION
**Pull request by amark-axcient**
npm-audit-action currently rejects the `audit-level` argument of `none` with `Error: Invalid input: audit_level`, when `none` is a valid argument

Please see https://docs.npmjs.com/cli/v8/commands/npm-audit#audit-level

While the argument is not well documented, I have found it will generate a report without an error exit code as discussed here:
https://github.com/npm/cli/issues/1182

Please Note: I'm not sure why dist/index.js has such a large delta - I expected a single line change. In the absence of specific contribution instructions I ran `npm run all`

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export async function run(): Promise<void> {
 
     // get audit-level
     const auditLevel = core.getInput('audit_level', {required: true})
-    if (!['critical', 'high', 'moderate', 'low'].includes(auditLevel)) {
+    if (!['critical', 'high', 'moderate', 'low', 'none'].includes(auditLevel)) {
```
Comment (oke-py) [inline src/main.ts:24]:
```suggestion
    if (!['critical', 'high', 'moderate', 'low', 'info', 'none'].includes(auditLevel)) {
```
The documentation you told me also includes "info".
https://docs.npmjs.com/cli/v8/commands/npm-audit#audit-level

**Review by oke-py — CHANGES_REQUESTED** [review]
Thanks!
As for `npm run all`, something is wrong. I don't know why. Could you run it again?

In my environment, `npm run all` causes this diff.
```diff
diff --git a/dist/index.js b/dist/index.js
index 736da46..a5bcf59 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -187,7 +187,7 @@ function run() {
             core.info(`Current working directory: ${process.cwd()}`);
             // get audit-level
             const auditLevel = core.getInput('audit_level', { required: true });
-            if (!['critical', 'high', 'moderate', 'low'].includes(auditLevel)) {
+            if (!['critical', 'high', 'moderate', 'low', 'info', 'none'].includes(auditLevel)) {
                 throw new Error('Invalid input: audit_level');
             }
             const productionFlag = core.getInput('production_flag', { required: false });
diff --git a/src/main.ts b/src/main.ts
index 71eeab2..bac4272 100644
--- a/src/main.ts
+++ b/src/main.ts
@@ -21,7 +21,11 @@ export async function run(): Promise<void> {
 
     // get audit-level
     const auditLevel = core.getInput('audit_level', {required: true})
-    if (!['critical', 'high', 'moderate', 'low'].includes(auditLevel)) {
+    if (
+      !['critical', 'high', 'moderate', 'low', 'info', 'none'].includes(
+        auditLevel
+      )
+    ) {
       throw new Error('Invalid input: audit_level')
     }
 ```

**Comment by amark-axcient** [comment]
@oke-py I have tried running it about 6 times across 3 fresh feature branches. I reran `npm ci`. I started new feature branches. I get similar results each time, thousands of lines change in the index.js, even if I manually update the files without running `npm run all`, when I push the branch up and compare it there are thousands of lines of changes in the index.js. My apologies, I _tried_ to treat this as a contribution rather than a feature request, but I don't know what is happening in that index.js so I'm going to close this. Thank you for your work on this GitHub Action.

**Comment by oke-py** [comment]
@amark-axcient Thanks for your idea and challenge.
I edited the code and released `v2.1.0` tag and associated `v2` tag with `v2.1.0`.

**Pull request closed without merging**